### PR TITLE
Minor Docker cleanups

### PR DESCRIPTION
**Pull request by md5**
* <s>Fix typo in `COPY` command</s>
* Add `-f` to Curl call to fail on HTTP codes >= 400
* Add `set -e` to entrypoint script to fail on errors
* <s>Change `echo` commands in entrypoint to echo to STDERR</s>
* <s>Make entrypoint whitespace consistent</s>

### CODE REVIEW

**Comment by md5** [comment]
Rebased after https://github.com/cloudbees/jenkins-ci.org-docker/commit/66dc63923438e70eaebd68da41bc7b9ea35ce2aa

File: Dockerfile [hunk 1]
```diff
@@ -19,13 +19,13 @@ VOLUME /var/jenkins_home
 RUN mkdir -p /usr/share/jenkins/ref/init.groovy.d
 
 
-COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-angent-port.groovy
+COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-agent-port.groovy
```
Comment (jglick) [inline Dockerfile]: Already there, merge with `master`.

File: Dockerfile [hunk 2]
```diff
@@ -19,13 +19,13 @@ VOLUME /var/jenkins_home
...
 
 ENV JENKINS_VERSION 1.596.2
 
 # could use ADD but this one does not check Last-Modified header 
 # see https://github.com/docker/docker/issues/8331
-RUN curl -L http://mirrors.jenkins-ci.org/war-stable/$JENKINS_VERSION/jenkins.war -o /usr/share/jenkins/jenkins.war
+RUN curl -fL http://mirrors.jenkins-ci.org/war-stable/$JENKINS_VERSION/jenkins.war -o /usr/share/jenkins/jenkins.war
```
Comment (jglick) [inline Dockerfile]: See #81.

File: jenkins.sh [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
-	f=${1%/} 
-	echo "$f"
-    rel=${f:23}
-    dir=$(dirname ${f})
-    echo " $f -> $rel"    
+	f=${1%/}
+	echo >&2 "$f"
+	rel=${f:23}
+	dir=$(dirname ${f})
+	echo >&2 " $f -> $rel"
```
Comment (jglick) [inline jenkins.sh]: Clashes with #89.

File: Dockerfile (hunk 2 above)
Comment (md5) [inline Dockerfile]: Shouldn't #81 see this one? :stuck_out_tongue:

File: Dockerfile (hunk 1 above)
Comment (md5) [inline Dockerfile]: I'll rebase again.

File: jenkins.sh (hunk 3 above)
Comment (md5) [inline jenkins.sh]: If you want to merge #89 first, I can rebase again after that.

**Comment by md5** [comment]
Rebased again.

**Comment by carlossg** [comment]
thanks!

**Comment by md5** [comment]
:metal:

**Pull request merged**